scripts/ci-is-vendor-in-sync.sh

```diff
@@ -17,6 +17,7 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+set -o verbose
```

Comment:
this change seems out of scope of the PR?

Reply:
the rest is LGTM.

Reply:
You're correct, that was a leftover from testing

Reply:
Fixed now